docs/code/getting-started.md

```diff
@@ -422,6 +422,40 @@ Let's review what all we've learned to do:
 
 That's a lot! You're well on your way to becoming a bonified app developer for elementary OS. Give yourself a pat on the back, then take some time to play around with this example. Change the names of files and see if you can still build and install them properly. Ask another developer to branch your project from launchpad and see if it builds and installs cleanly on their computer. If so, you've just distributed your first app! When you're ready, we'll move onto the next section: Packaging.
 
+# Adding Translations {#Adding-Translations}
+Now that you've learn about cmake, the next step is to make your app able to be translated to different languages. The first thing you need to know is how to convert strings in your code into translatable strings.
+
+        stdout.printf ("Normal String");
```

Comment:
Instead of normal, we should call this "not translatable" to make sure we're giving it a negative connotation